src/expr.rs

```diff
@@ -35,7 +37,222 @@ pub enum Expr {
     Geometry(Geometry),
 }
 
+impl From<Value> for Expr {
+    fn from(v: Value) -> Expr {
+        let e: Expr = serde_json::from_value(v).unwrap();
```

Comment:
Nit: If there's an `unwrap` in there, you usually should be implementing `TryFrom` instead of `From`.